### PR TITLE
Resolve some Vagrant issues

### PR DESCRIPTION
**Pull request by thebeline**
Please see Commit Messages for explanations.  Also, there are come Git Commands that accompany these changes for Windows hosts.  These changes will be in-bound.

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -49,13 +49,17 @@ Vagrant.configure("2") do |config|
   config.vm.provider "virtualbox" do |box|
     box.memory = 1024
     box.cpus = 1
+    box.customize ["setextradata", :id, "VBoxInternal2/SharedFoldersEnableSymlinksCreate/v-root", "1"]
   end
 
+  config.vm.provision :shell, :inline => 'apt-get update'
```
Comment (sudo-reboot) [inline Vagrantfile]: this shouldn't be done here, it belongs in the provisioning code for the project. fairly certain the startupgenome chef code you're working with does this first thing, but if it doesn't it should

File: Vagrantfile [hunk 2]
```diff
@@ -49,13 +49,17 @@ Vagrant.configure("2") do |config|
   config.vm.provider "virtualbox" do |box|
     box.memory = 1024
     box.cpus = 1
+    box.customize ["setextradata", :id, "VBoxInternal2/SharedFoldersEnableSymlinksCreate/v-root", "1"]
```
Comment (sudo-reboot) [inline Vagrantfile]: will this explode on non-windows hosts?

**Comment by sudo-reboot** [comment]
what https validation errors are you getting? dev environments generally just don't try to use https

**Comment by thebeline** [comment]
Just wanted to let you know that I have received your comments, and I will get you examples and explanations shortly.

It would seem that your development team has not experienced some of the issues we have, and this would seem to largely be caused by the fact that we are primarily a Windows development team (Windows workstations, not servers) and you are primarily (exclusively?) OS X.

I will get you examples of the errors that each line resolves.  Also, I am going to close this PR (again) as I have moved things around again.  I will not open until it is frozen, I apologize for the confusion.

File: Vagrantfile (hunk 1 above)
Comment (thebeline) [inline Vagrantfile]: `default.rd` ran `include_recipe "git"` before `"apt"`, which was causing the problem (apt was hitting incorrect locations and yelling a bit/bunch).

I am very new to Vagrant, so I was unaware that simply swapping this order would resolve the issue.  Seems it does, so I will push that and remove this shell command.

File: Vagrantfile (hunk 2 above)
Comment (thebeline) [inline Vagrantfile]: We'll find out today.

**Comment by thebeline** [comment]
Https errors were during provisioning.

**Pull request closed without merging**